package exception;

public class ExceptionRunner {

	public static void main(String[] args) {
		method1();
		System.out.println("Main Ended");
	}
	private static void method1() {
		method2();
		System.out.println("Method1 Ended");
	}

	private static void method2() {
		String str = null;
		str.length(); // 널포인트 익셉션
		System.out.println("Method2 Ended");
	}

}
